frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts

```diff
@@ -291,46 +303,38 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
     this.buttons = [this.newNotebookButton];
   }
 
-  public pvcTrackByFn(index: number, pvc: VolumeProcessedObject) {
-    return `${pvc.name}/${pvc.namespace}/${pvc.size}`;
+  public pvcTrackByFn(index: number, pvc: PVCProcessedObject) {
+    return `${pvc.name}/${pvc.namespace}/${pvc.capacity}`;
   }
 
-  public parseIncomingData(pvcs: VolumeResponseObject[], notebooks: NotebookResponseObject[]) {
-    let pvcsCopy = JSON.parse(JSON.stringify(pvcs)) as VolumeProcessedObject[];
-    //Check which notebooks are mounted
-    let mounts = Object.fromEntries(
-      notebooks.flatMap(nb => nb.volumes.map(v => [v,nb]))
-    );
-
-    pvcsCopy.forEach(element => {
-      if(mounts[element.name]){
-        element.usedBy = mounts[element.name].name;
-        element.status = {} as Status;
-        element.status.message = $localize`Attached`;
-        element.status.phase = STATUS_TYPE.MOUNTED;
-      } else {
-        element.status = {} as Status;
-        element.status.message = $localize`Unattached`;
-        element.status.phase = STATUS_TYPE.UNMOUNTED;
-      }
```

Comment:
Attached vs unattached is ours

Reply:
yup. Thanks. Added that back in. With the update, there is a status field that should be coming from the backend. But i think in our custom case we just want attached or unattached.